src/main/java/pl/arturzgodka/controllers/HeroClassesAndSkillsLists.java

```diff
@@ -0,0 +1,34 @@
+package pl.arturzgodka.controllers;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class HeroClassesAndSkillsLists {
```

Comment:
Will you be using those lists for other purpose than creating a map in SkillsController? If not, then maybe it makes sense to create a public field with the map and make all those lists private

Reply:
done